UKHomeOffice/sgmr-service
# SPCR-390 Refactor check your answers

### AC / Description of changes
Refactor the current checkAnswers page to look like the prototype but not including the elements that require BE updates

Departure details (not including time to/from or country)
Arrival details (not including time to/from or country)
Pleasure craft details (use current fields until BE updates)
People on board (new table)
Skipper Details (will be the same as current version for now)
Employed people table

### To Test
- Create a voyage
- Fill in the departure form
- Click save and exit
- Return to the draft voyage
- > You should be taken to the check your answers page with all sections blank except departure details
- Click accept and submit voyage plan
- > All blank sections should error
- > Error links should scroll to the correct section
- Click the change link and add details for each section
- Change today's value in date.js file to a future date that you know is later than at least one of your saved peoples expiry dates
![image](https://user-images.githubusercontent.com/72464825/168840781-0b663bed-a1c7-4f29-a2c1-0bce90c940c1.png)
- Go to the CYA page
- Click accept and submit voyage plan
- > There should be an error when any documents are expired
- Revert the code in date.js
- Navigate to the CYA page
- Click accept and submit voyage plan
- If all details are correct, the form should submit and you should receive a confirmation email

### Notes

---

**Remember to merge to feature branches not `master`.**
